doc/design-authority/dhcw/architecture-decision-record-process/README.md

```diff
@@ -0,0 +1,168 @@
+# Architecture Decision Record (ADR) Process
+
+This document outlines the process for proposing, developing, collaborating
+on, and approving Architecture Decision Records (ADRs) within DHCW. The
+process emphasises collaboration and working in the open whilst minimising
+the involvement of formal governance bodies (e.g. Technical Design Authority
+(TDA) and Technical Design Assurance Group (TDAG)) as much as is practicable
+whilst maintaining suitable levels of assurance and governance.
+
+## Overview
+
+Anyone can propose an ADR and request collaboration to approve a decision.
+The proposal should clearly articulate the problem and context, and may
+include a solution or decision if ready. Early engagement is strongly
+encouraged, so it is expected proposals will be incomplete and very draft
+when first proposed.
+
+There are multiple levels of decision defined, which follow different
+processes proportionate to their impact/significance.
+
+The approved [ADR template](../architecture-decision-record-template.md)
+should be used to ensure consistency and completeness.
+
+The default is to collaborate on ADRs via the DHCW public GitHub repository,
+in the open. ADRs can be discussed in private by exception if the topic area
+is highly sensitive/impacts security etc.
+
+## Initiation Process
+
+Prior to the creation of an ADR it is recommended that an Issue is raised in
```

Comment:
Add link to https://github.com/GIG-Cymru-NHS-Wales/Architecture-Decision-Records/issues